daemons/attrd/attrd_attributes.c

```diff
@@ -102,16 +102,16 @@ GHashTable *attributes = NULL;
  * \internal
  * \brief Create an XML representation of an attribute for use in peer messages
  *
- * \param[in] parent       Create attribute XML as child element of this element
- * \param[in] a            Attribute to represent
- * \param[in] v            Attribute value to represent
- * \param[in] force_write  If true, value should be written even if unchanged
+ * \param[in,out] parent      Create attribute XML as child element of this
+ * \param[in]     a           Attribute to represent
+ * \param[in]     v           Attribute value to represent
+ * \param[in]     force_write If true, value should be written even if unchanged
  *
```

Comment:
^ if you feel like keeping two spaces after `force_write` for consistency. Wouldn't bother unless you're updating this for another reason

Reply:
Also to keep it under 80